src/modules/builtin/echo.rs

```diff
@@ -1,4 +1,5 @@
 use heraclitus_compiler::prelude::*;
+use crate::docs::module::DocumentationModule;
```

Comment:
We need to add that stuff on the contributing guide

Reply:
I'll write this in this PR